Cleanup `Beatmap/Packs` (#9596)
ppy/osu-wiki: wiki/Beatmap/Packs/en.md

outdated: true
---

<!-- TODO: new mirrors -->

# Beatmap packs

A beatmaps pack is a compressed file containing beatmaps. It is worth noting that the most packs are:

> - Roughly either ~100MB (Taiko/osu!mania) or ~200MB (-/Approved) <!-- TODO: Classifying like this looks inappropriate -->

Reviewer: maybe @peppy or someone else could find an average and/or range of file sizes if that's relevant to mention

- Contain videos (if included)

## Naming

> The filename of the pack is based on its content, category, and upload order. For example, a beatmap pack file named:

Reviewer: ```suggestion
The filename of the pack is based on its content, category, and upload order. For example, in a beatmap pack file named:
```

```
F2 - Rohi Pack.zip
```

> `F2` means the second beatmap pack under the Featured Artist category, while `Rohi Pack` refers to the pack's name shown on the beatmap packs website. This pack includes Rohi's songs only.

Reviewer: ```suggestion
`F2` means the second beatmap pack under the Featured Artist category, while `Rohi Pack` refers to the pack's name shown on the [beatmap pack listing](https://osu.ppy.sh/beatmaps/packs/F2). This pack includes [Rohi](https://osu.ppy.sh/beatmaps/artists/82)'s songs only.
```

Sometimes numbers exist in beatmap pack names, especially for Standard packs. As beatmaps in most of them are just close in time, thus using numbers to indicate their upload order. For example, `Approved Beatmap Pack #7` contains only  [approved](/wiki/Beatmap/Category#approved) beatmaps, and it is the 7th uploaded beatmap pack.

## Categories

Beatmaps packs are split into 7 categories, with a special letter indicator:

| Indicator | Category | Beatmap types |
| :-: | :-- | :-- |
| **S** | Standard | Ranked or Approved beatmaps |

> | **F** | Featured Artist | High-quality songs from the [Featured Artists](/wiki/People/Featured_Artists) |

Reviewer: subjective wording should be left out

```suggestion
| **F** | Featured Artist | Songs from [Featured Artists](/wiki/People/Featured_Artists) |
```

> | **T** | Tournament | Tournament-specific beatmaps <!-- TODO: Uncertain about its real use--> |

Reviewer: ```suggestion
| **T** | Tournament | Tournament mappools |
```

| **L** | Project Loved | Beatmaps chosen by the community in [Project Loved](/wiki/Community/Project_Loved) |

> | **R** | Spotlights | Seasonal beatmap challenge packs by [Beatmap Spotlights](wiki/Beatmap_Spotlights), formly known as **Charts** |

Reviewer: ```suggestion
| **R** | Spotlights | Seasonal beatmap challenge packs from the [Beatmap Spotlights](/wiki/Beatmap_Spotlights), formerly known as *Charts* |
```

> | **T** | Theme | Theme based beatmaps |
> | **A** | Artist/Album | Artist/Album based beatmaps |

Reviewer: ```suggestion
| **T** | Theme | Beatmaps based on a particular theme |
| **A** | Artist/Album | Songs from specific artists or albums |
```
